uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 2
+revision = 3
```

Comment:
@LakshyAAAgrawal I'm aware my uv (version 0.8.14) updated the revision here.

Let me know if you prefer me to revert it.